### PR TITLE
Lca classifier

### PR DESCRIPTION
**Pull request by HassanAmr**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/lcaclassifier/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package:
+  name: lcaclassifier
+  version: 3.0.7
+
+source:
+  url: https://github.com/lanzen/CREST/releases/download/3.0.7/LCAClassifierV3.0.7.tar.gz
+  md5: ca072806b9251f280978c6a3ba520395
```
Comment (mbargull) [inline recipes/lcaclassifier/meta.yaml]: Please use `sha256` instead.

File: recipes/lcaclassifier/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+build:
+  skip: True  # [py3k]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biom-format
```
Comment (mbargull) [inline recipes/lcaclassifier/meta.yaml]: Only `python` and `setuptools` are required for the build; you can remove the other two.

File: recipes/lcaclassifier/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - biopython
+
+  run:
+    - python
+    - biom-format
+    - biopython
+
+test:
+  commands:
+    - classify -h
```
Comment (mbargull) [inline recipes/lcaclassifier/meta.yaml]: We can also test the other entrypoint, i.e.,
```yaml
    - nds2CREST -h
```

File: recipes/lcaclassifier/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+source:
+  url: https://github.com/lanzen/CREST/releases/download/3.0.7/LCAClassifierV3.0.7.tar.gz
+  md5: ca072806b9251f280978c6a3ba520395
+  patches:
+    - lcaclassifier.patch
+
+build:
+  skip: True  # [py3k]
+  number: 0
```
Comment (mbargull) [inline recipes/lcaclassifier/meta.yaml]: Can you add
```yaml
  entry_points:
    - classify = LCAClassifier.classify:main
    - nds2CREST = LCAClassifier.nds2CREST:main
```
to the `build` section?

File: recipes/lcaclassifier/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biom-format
+    - biopython
+
+  run:
+    - python
```
Comment (mbargull) [inline recipes/lcaclassifier/meta.yaml]: Acc. to `setup.py`, `setuptools` is also a run requirement.
But I can't see why, so you can either
1. add a patch to remove `setuptools` from `install_requires` in `setup.py` (preferred), or
2. add add `setuptools` to the run requirements

File: recipes/lcaclassifier/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+package:
+  name: lcaclassifier
+  version: 3.0.7
+
+source:
+  url: https://github.com/lanzen/CREST/releases/download/3.0.7/LCAClassifierV3.0.7.tar.gz
+  md5: ca072806b9251f280978c6a3ba520395
+  patches:
+    - lcaclassifier.patch
```
Comment (mbargull) [inline recipes/lcaclassifier/meta.yaml]: Please remove that patch.

File: recipes/lcaclassifier/lcaclassifier.patch [hunk 7]
```diff
@@ -0,0 +1,100 @@
+diff -u -r a/LCAClassifier/src/LCAClassifier/arbor.py b/LCAClassifier/src/LCAClassifier/arbor.py
```
Comment (mbargull) [inline recipes/lcaclassifier/lcaclassifier.patch]: Please remove this patch file. It doesn't add anything valuable as it only adds shebangs to files that end up in `$PREFIX/lib`.

File: recipes/lcaclassifier/build.sh [hunk 8]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (mbargull) [inline recipes/lcaclassifier/build.sh]: Looks like the software is meant to be installed with `install.sh` instead of using `setup.py` directly.

Comment (HassanAmr) [reply recipes/lcaclassifier/build.sh]: I will need `buildout`/`zc.buildout' in that case, so, I will add package for it to conda-forge maybe. Thanks.

**Comment by daler** [comment]
Closing due to inactivity., please re-open if needed.

**Pull request closed without merging**